app/service/account.py

```diff
@@ -0,0 +1,17 @@
+from fastapi import Depends
+from jose import jwt
+
+from app.models.models import Account
+from app.service.auth import oauth2_scheme, SECRET_KEY, ALGORITHM, session, get_current_user
+
+
+async def create_account(account: Account, user: dict = Depends(get_current_user)) -> list[Account]:
+    new_account = Account()
+    new_account.price = account.price
+    new_account.memo = account.memo
+    new_account.user_id = user.get("user_id")
```

Comment:
P2: Account 객체 생성시 넣을 수 있는 필드는 생성자에서 파라미터로 전달해서 생성하는 편이 깔끔해서 좋을 것 같아요.
```suggestion
    user_id = user.get("user_id")
    new_account = Account(price=account.price, memo=account.memo, user_id=user_id)
```